src/ZenstruckFoundryBundle.php

```diff
@@ -28,7 +28,7 @@ final class ZenstruckFoundryBundle extends Bundle
 {
     public function boot(): void
     {
-        if (!Factory::isBooted()) {
+        if (!Factory::isBooted() && $this->container) {
```

Comment:
fixes a minor test in sca due to this change
https://github.com/symfony/symfony/commit/7a8457df3a2507fa4e254622847b429d5c67d733